src/constants.js

```diff
@@ -1,37 +1,26 @@
-export const mockedCoursesList = [
-  {
-    id: 'de5aaa59-90f5-4dbc-b8a9-aaf205c551ba',
-    title: 'JavaScript',
-    description: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.`,
-    creationDate: '8/3/2021',
-    duration: 160,
-    authors: ['27cc3006-e93a-4748-8ca8-73d06aa93b6d', 'f762978b-61eb-4096-812b-ebde22838167'],
-  },
-  {
-    id: 'b5630fdd-7bf7-4d39-b75a-2b5906fd0916',
-    title: 'Angular',
-    description: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.`,
-    creationDate: '10/11/2020',
-    duration: 210,
-    authors: ['df32994e-b23d-497c-9e4d-84e4dc02882f', '095a1817-d45b-4ed7-9cf7-b2417bcbf748'],
-  },
-];
+// constants.js
+export const api_endpoints = {
+  courses: 'http://localhost:4000/courses/all',
```

Comment:
It's better to omit the origin of `http://localhost:4000` and have the constants point to `/courses/all` etc. instead. That way our constants don't hardcode the environment origin themselves. Then you can have another constant for current environment and append them `API_ENDPOINT.someEndpoint` to it.